src/scss/components/_OptionList.scss

```diff
@@ -0,0 +1,53 @@
+/*Option List*/
+
+.OptionList{
+  display: inline-block;
+  justify-content: center;
+  width: 25%;
+  color: $defaultFontColor;
+
+  a{
+    text-decoration: none;
+    cursor: pointer;
+    color: $defaultFontColor;
+  }
+
+  .productListItem{
```

Comment:
w tym pliku powinny być style tylko do ProductList
to co tyczy osobnego komponentu Product (w Twoim przypadku .productListItem) powinno mieścić się w osobnym pliku  scss